src/openforms/forms/tests/test_api.py

```diff
@@ -157,8 +162,24 @@ def test_create_form_unsuccessful_with_bad_data(self):
         self.assertEqual(
             response.json(),
             {
-                "name": [_("This field is required.")],
-                "slug": [_("This field is required.")],
+                "type": "http://testserver/exception-handler/fouten/ValidationError/",
+                "code": "invalid",
+                "title": "Invalid input.",
```

Comment:
These should also be marked as translatable and go through the translation machinery.